includes/class-freemius.php

```diff
@@ -24245,6 +23895,20 @@ private function maybe_modify_api_curl_error_message( $result ) {
                 );
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.5.4
+         *
+         * @param object $result
+         */
+        private function maybe_modify_api_blocked_error_message( $result ) {

```

Comment:
@fajardoleo I'm all up for code reusability, but since it's only used 2 times and doesn't encapsulate much logic, I think it will be better to use the following instead explicitly:
```php
if ( FS_Api::is_blocked( $result ) ) {
    $result->error->message = $this->generate_api_blocked_notice_message_from_result( $result );
}
```

Also, it will produce fewer lines of code  (even without counting the PHP-Doc).